kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -63,6 +63,11 @@
   public static final String REGION_CONFIG = "s3.region";
   public static final String REGION_DEFAULT = Regions.DEFAULT_REGION.getName();
 
+  public static final String S3_PROXY_HOST = "s3.proxy.host";
```

Comment:
@kkonstantine Using just a URL has its own problems, e.g. that the password still needs to be added as a separate field to avoid leaking secrets in log files (or you just need to make the entire URL a `PASSWORD` field).

I don't know much about how people use proxies for S3 in practice, so I'm not sure which features listed on that page are important. For example, besides host and port, I could imagine that SSL might be required for something like this.